core/src/main/java/org/apache/brooklyn/core/location/Machines.java

```diff
@@ -102,15 +100,11 @@
     }
     
     public static Maybe<MachineLocation> findUniqueMachineLocation(Iterable<? extends Location> locations) {
-        return findUniqueElement(locations, MachineLocation.class);
+        return findUniqueMachineLocation(locations, MachineLocation.class);
     }
 
-    public static Maybe<SshMachineLocation> findUniqueSshMachineLocation(Iterable<? extends Location> locations) {
```

Comment:
We shouldn't delete things without first deprecating. I'll add this back in, and deprecate it.